README.md

```diff
@@ -130,7 +130,7 @@ Fork `master` branch into your personal repository. Clone it to local computer.
 ```sh
 $ git clone https://github.com/{your-personal-repo}/tui.editor.git
 $ npm install
-$ cd apps/editor
+$ cd tui.editor/apps/editor
```

Comment:
I don't think you should modify it like this. The project name can be modified depending on the user, so I think it should be specified in a different way. After the clone command, a command to move to the project folder should be added. It would be nice to add a command to move to the project folder after the clone command. And how about specifying `tui.editor` as the default value of `[project-name]`?

```diff
$ git clone https://github.com/{your-personal-repo}/tui.editor.git
+ cd [project-name]
$ npm install
- $ cd tui.editor/apps/editor
+ $ cd [project-name]/apps/editor
```